oscillogram_classification/cam.py

```diff
@@ -254,20 +256,21 @@ def gen_heatmaps_overlay_side_by_side(cams: dict, voltage_vals: np.ndarray, titl
             cams[list(cams.keys())[i]][np.newaxis, :], cmap="plasma", aspect="auto", alpha=.75, extent=extent
         )
         # data
-        axes[i].plot(data_points, voltage_vals, '#000000')
+        axes[i].plot(time_vals, voltage_vals, '#000000')
     plt.tight_layout()
 
 
-def gen_heatmaps_as_overlay(cams: dict, voltage_vals: np.ndarray, title: str) -> Image:
+def gen_heatmaps_as_overlay(cams: dict, voltage_vals: np.ndarray, title: str, time_vals: List[float]) -> Image:
```

Comment:
It's generally not a good idea to change the signature of functions that are used in the other repositories, it will stop being compatible with all the usages. In this case, since it probably makes sense to always return these plots with seconds on the y-axis, I tend to still accept it and change the function calls in `vehicle_diag_smach`. Already opened an issue for this.

Reply:
Better option would be to do just as you did in the preprocessing below, but as said, in this case it is fine.